game.js

```diff
@@ -56,6 +39,11 @@ Player2.prototype.placeMarker = function(){
 		console.log("invalid input");
 		selectPosition();
 	}
+};
+//computer select position
+ComputerPlayer.prototype.placeMarker = function(board){
+	// AI logic for placing markers goes here.
+	// 
```

Comment:
A computer player implements the same function as HumanPlayer, but the computer has to make a choice. You could implement this most simply by taking a Random number between 1-9. You will have to decide if you want validation that the choice isn't taken already inside of the `player` or inside of the `Game`.

It might make the most sense to just return the choice, and have the game call this method again if the choice is invalid (in some kind of loop that stops asking [calling placeMarker] once the choice is valid)